css.css

```diff
@@ -0,0 +1,128 @@
+body {
+  background: rgb(188,86,188);
+background: radial-gradient(circle, rgba(188,86,188,1) 0%, rgba(150,182,255,1) 100%);
+}
+.container {
+  width: 1200px;
+  height: 100%;
+  margin: auto;
+  padding-top: 30px;
+}
+
+/* Search */
+.search {
+
+}
+
+.search h2 {
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+
+
+/* Tests */
+#TestsAndResult {
+
+}
+
+.wrapper {
+  display: flex;
+}
+
+/* Arguments */
+#tests {
+  margin-right: 40px;
+}
+
+#testsList {
+  padding-left: 3px;
+  direction:rtl;
+  height: 500px;
+  width: 400px;
+  background-color: white;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  /* white-space: nowrap; */
+}
+.testCase {
+  direction: ltr;
+}
+
+.testName {
+  height: 30px;
+  width: 400px;
+  display: flex;
+  background-color: white;
+  vertical-align: middle;
+  box-sizing: border-box;
+  border-bottom: 1px solid #44CFCB;
+}
+
+.test {
+  border-bottom: 1px solid purple;
+  width: 100%;
+  background-color: #55825d;
+  display: none;
+}
+
+.createTestCaseButton {
+ display: none;
+}
+input[type="text"] {
+    background: transparent;
+    border: none;
```

Comment:
I don't know whether input is still visible after that lines, but looks pretty weird.

Reply:
Yeah
@Co11y could you explain what you want to do with this code?